content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,180 @@
+
+4. Choose a variable set scope:
+   - **Apply to all workspaces in this organization:** Terraform Cloud will automatically apply this global variable set to all existing and future workspaces.
+   - **Apply to specific workspaces:** Use the text field to search for and select one or more workspaces where Terraform Cloud should apply this variable set.
+
+
+5. Add one or more variables: Click **+ Add variable**, choose a variable category (Terraform or environment), optionally mark the variable as [sensitive](#sensitive-values), and enter a variable name, value, and optional description, and click **Save variable**.
+
+    Refer to [variable values and format](#variable-values-and-format) for variable limits, allowable values, and formatting.
+
+    ~> **Warning:** Be careful when duplicating existing variables. Terraform Cloud will not create this variable set if you selected **Apply to all workspaces in this organization** and you add one or more variables with the same key as other global variable sets.
+
+6. Click **Create variable set.** Terraform Cloud adds the new variable set to any specified workspaces and displays it on the **Variable Sets** page.
+
+### Edit Variable Sets
+
+To edit or remove variable sets:
+
+1. Click **Settings** in the top menu bar and then click **Variable Sets** in the left sidebar. The **Create a new Variable set** page appears.
+2. Click the variable set you want to edit. The **Edit Variables** page appears, where you can change the variable set settings. Refer to [create variable sets](#create-variable-sets) for details.
+
+### Delete Variable Sets
+
+To delete the variable set:
+
+1. Click **Settings** in the top menu bar and then click **Variable Sets** in the left sidebar. The **Create a new Variable set** page appears.
+2. Select **Delete variable set**. Terraform Cloud removes the variable set from all workspaces.
+
+### Apply or Remove Variable Sets From Inside a Workspace
+
+To apply a variable set to a specific workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+
+2. Click **Apply Variable Set**, select the variable set you want to apply to your workspace, and click **Apply variable set**. The new variable set appears in the list and the Terraform will use the variables in the set during the next workspace run.
+
+To remove a variable set from within a workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+2. Click the ellipses button next to the variable set and select **Remove variable set**.
+3. Click **Remove variable set** in the dialog box. Terraform Cloud removes the variable set from this workspace, but it remains available to other workspaces in the organization.
```

Comment:
This is getting updated so that removal entails clicking through to the variable set detail page and removing from there. My understanding is we don't have a good pattern around this elsewhere? @roarytubbs Has better context.

Reply:
This will be changing for if a user has permissions they will just click on the variable set "card" and it will take them to view/edit. I don't think that will get changed before we launch into public beta tomorrow.

Reply:
okay then I'm going to leave as is, and we can fix as soon as it gets implemented - thanks for the heads up!